src/commentsDashboard/index.jsx

```diff
@@ -0,0 +1,71 @@
+import PostsList from "../postsList/";
+import { useCommentsDashboard } from "./useCommentsDashboard.js";
+import { Button, ButtonGroup } from "@mui/material";
+import PhotosList from "../photosList/index.jsx";
+
+const DisplayMode = {
+  posts: "Browse posts:",
+  photos: "Browse photos:",
+};
+
+export const CommentsDashboard = () => {
+  const {
+    users,
+    isLoading,
+    activeUser,
+    displayMode,
+    setNewActiveUser,
+    switchDisplayModeToPhotos,
+    switchDisplayModeToPosts,
+  } = useCommentsDashboard();
+
+  return (
+    <>
+      <div>
+        <Button
+          variant={"contained"}
+          color={displayMode === DisplayMode.posts ? "success" : ""}
+          onClick={switchDisplayModeToPosts}
+        >
+          Posts
+        </Button>
+        <Button
+          variant={"contained"}
+          color={displayMode === DisplayMode.photos ? "success" : ""}
+          onClick={switchDisplayModeToPhotos}
+        >
+          Photos
+        </Button>
+      </div>
+
+      <h1>
+        {displayMode === DisplayMode.posts
+          ? DisplayMode.posts
+          : DisplayMode.photos}
+      </h1>
+      {isLoading && <p>Loading...</p>}
```

Comment:
I already pointed out to use ternary operator for such a cases

Reply:
done